src/languages/scala.js

```diff
@@ -0,0 +1,94 @@
+import { Comments, Helpers } from "../utils";
```

Comment:
You'll need to add the .js to the end of these imports - please follow what has been done in other files.